DEVELOPMENT.md

```diff
@@ -153,8 +153,11 @@ By default, KubeArchive listens to `Event`s in the `test` namespace.
     kubectl get -n kubearchive secrets kubearchive-api-server-tls -o jsonpath='{.data.ca\.crt}' | base64 -d > ca.crt
     ```
 
-1. **[ Optional ]** Create a service account with a specific role to test the REST API.
-    This Helm chart already provides `kubearchive-test-sa` with `view` privileges for testing purposes.
+1. Create a service account with a specific role to test the REST API.
+   You can use the `default` user with `view` privileges provided for the `test` namespace in `test/users/test-user.yaml`
```

Comment:
You need a period at the end of the sentence:

```suggestion
   You can use the `default` user with `view` privileges provided for the `test` namespace in `test/users/test-user.yaml`.
```

Reply:
Fixed